core/src/main/java/org/jdbi/v3/core/mapper/reflect/BeanMapper.java

```diff
@@ -186,10 +186,10 @@ public T map(ResultSet rs, StatementContext ctx) throws SQLException {
                             .map(m -> m.getParameterCount() > 0 ? m.getParameters()[0] : null)
                             .orElse(null);
 
-                        QualifiedType type = QualifiedType.of(propertyType(descriptor))
+                        QualifiedType<?> type = QualifiedType.of(propertyType(descriptor))
                             .with(getQualifiers(getter, setter, setterParam));
                         ColumnMapper<?> mapper = ctx.findColumnMapperFor(type)
-                            .orElse((r, n, c) -> r.getObject(n));
+                            .orElse((ColumnMapper) (r, n, c) -> r.getObject(n));
```

Comment:
This probably wants a `@SuppressWarnings("raw")`

Reply:
I don't see this as actually resolved.  There's an `unchecked` suppression but not one for `rawtypes`.